### DISCUSSION TITLE
InsertRange to ObservableRangeCollection

### DISCUSSION THREAD

**Discussion by acaliaro** [CommunityToolkit/Maui#135] [New Feature Discussions]
Now ObservableRangeCollection has an AddRange method. Is it possible to have also an InsertRange method with a "index" parameter like "List" has?

**Comment by TheCodeTraveler** [comment]
Hi @acaliaro!

We [won't be adding ObservableRangeCollection](https://github.com/CommunityToolkit/Maui/issues/99) to the .NET MAUI Community Toolkit. We are derring all MVVM-focused tools to [`CommunityToolkit.MVVM`](https://www.nuget.org/packages/CommunityToolkit.Mvvm/).

I recommend opening a discussion on the MVVM Toolkit for this topic 👍

**Reply by acaliaro** [reply]
Is CommunityToolkit.MVVM ready for Xamarin / Maui?

**Reply by TheCodeTraveler** [reply]
Yup! The MVVM Toolkit it platform-independent, ie It's a .NET Standard library that does have any iOS/Android/UWP/WPF etc dependencies.

We will even be using the MVVM Toolkit in our sample app, `CommunityToolkit.Maui.Samples`: https://github.com/CommunityToolkit/Maui/issues/9

**Comment by CliffAgius** [comment]
Closed as Answered.
